test/integration/consumer_group_test.exs

```diff
@@ -100,10 +100,12 @@ defmodule KafkaEx.ConsumerGroup.Test do
   end
 
   test "fetch does not commit offset with auto_commit is set to false" do
-    KafkaEx.create_worker(:fetch_no_auto_commit_worker)
-    Enum.each(1..10, fn _ -> KafkaEx.produce(%Proto.Produce.Request{topic: "food", required_acks: 1, messages: [%Proto.Produce.Message{value: "hey"}]}) end)
-    offset = KafkaEx.fetch("food", 0, offset: 0, worker: :fetch_no_auto_commit_worker, auto_commit: false) |> hd |> Map.get(:partitions) |> hd |> Map.get(:message_set) |> Enum.reverse |> hd |> Map.get(:offset)
-    offset_fetch_response = KafkaEx.offset_fetch(:fetch_no_auto_commit_worker, %Proto.OffsetFetch.Request{topic: "food"}) |> hd
+    topic = generate_random_string
+    worker_name = :fetch_no_auto_commit_worker
+    KafkaEx.create_worker(worker_name)
+    Enum.each(1..10, fn _ -> KafkaEx.produce(%Proto.Produce.Request{topic: topic, required_acks: 1, messages: [%Proto.Produce.Message{value: "hey"}]}, worker_name: worker_name) end)
```

Comment:
added `worker_name` to the produce call and it made this test pass consistently. Without it, it fails consistently as mentioned in the related issue